dispatch/static/manager/src/js/components/inputs/ItemSelectInput.js

```diff
@@ -128,7 +129,7 @@ class ItemSelectInput extends React.Component {
           isSelected={false}
           text={item[this.props.attribute]}
           onClick={() => this.addValue(item.id)} />
-      ))
+      )) : []
```

Comment:
Instead of this you can add `results` to the defaultProps: https://github.com/ubyssey/dispatch/blob/3eb48b97ef296d098c7159f12e7d0965298d88b1/dispatch/static/manager/src/js/components/inputs/ItemSelectInput.js#L176-L178